### ISSUE TITLE
Unicode support in ExternalCodeEditor_WIN32.cxx

### ISSUE THREAD

**Issue by ZJUGKC** [fltk/fltk#453]
Several Windows API functions with `W` version should be used in `fluid/ExternalCodeEditor_WIN32.cxx`.

```
GetFileAttributeA -> GetFileAttributeW
PostMessage -> PostMessageW
CreateFile -> CreateFileW
DeleteFile -> DeleteFileW
GetTempPath -> GetTempPathW
RemoveDirectory -> RemoveDirectoryW
CreateDirectory -> CreateDirectoryW
CreateProcess -> CreateProcessW
```

**Comment by Albrecht-S** [comment]
Good point, thanks. This really needs to be done. We **must** always use the "wide character", i.e. Unicode versions of system functions which are in most cases wrapped in `fl_*` function names. I'm not sure if just renaming the function calls is enough though (_if this was the suggestion of the OP_) - this should be investigated and verified.

@erco77 Greg, ISTR that you are the original author of the `ExternalCodeEditor` functions. Would you like to take this issue?

**Comment by MatthiasWM** [comment]
PR #668

**Comment by Albrecht-S** [comment]
PR #668 looks fine, good to go from me.

**Comment by MatthiasWM** [comment]
Fixed in 59d3b2e9fd10bdf14592e82ced422346ecd7204e
